docs/src/app/page.md

```diff
@@ -2,15 +2,29 @@
 title: Welcome
 ---
 
-LaunchFlow enables you to manage your application's infrastructure, secrets, and deployments from Python code. It runs entirely on your local machine and everything is created in your own cloud account. Follow the [Get Started Guide](/docs/get-started) to start launching infrastructure on GCP, AWS, and Docker in a matter of minutes.
+LaunchFlow is an open source deployment tool that makes it easy to automate your application's infrastructure, secrets, and deployments on Amazon Web Services (AWS) and Google Cloud Platform (GCP).
+
+Use the Python SDK to define your infrastructure in code, then run `lf deploy` to deploy everything to a dedicated environment in your cloud account.
+
+LaunchFlow runs entirely on your local machine and everything is created in your own cloud account. Follow the [Get Started Guide](/docs/get-started) to launch an example API in minutes.
+
+
+{% callout type="note" %}
+**LaunchFlow is not just for deploying Python apps.**
+
+The Python SDK is used to define your infrastructure in code, but you can deploy any static or Dockerized application to AWS or GCP.
+
+Python is just the language for your DevOps automation.
+
+{% /callout %}
 
 ## Core Concepts
 
-[Resources](https://docs.launchflow.com/docs/concepts/resources) allow you to add things like databases, storage, and networking to your application by simply importing them in your code.
+[Services](/docs/concepts/services) allow you to deploy websites, APIs, workflows, and other types of applications to your cloud account with minimal setup. All you need to provide is a Dockerfile - LaunchFlow will take care of the rest.
 
-[Services](https://docs.launchflow.com/docs/concepts/services) make it easy to setup Docker registries, load balancers, service containers, and other configuration required to deploy and release your application.
+[Resources](/docs/concepts/resources) allow you to add things like databases, storage, and task queues to your application by simply importing them in your code.
```

Comment:
nit: maybe drop "things like" something like:

allow you to add databases, storage, task queues, and more

feel like that flows a little better